libs/solid/skeleton/src/lib/components/feedback/feedback.component.html

```diff
@@ -15,9 +15,14 @@ <h1 mat-dialog-title>{{ getTitle() }}</h1>
     <mat-form-field>
       <input matInput formControlName="subject" placeholder="Betreff" />
     </mat-form-field>
+    <div *ngIf="getLocation()">
+      <mat-label style="font-weight: bold; text-align: center"
```

Comment:
Bitte keine inline styles.
Styles gehören in die `*component.scss`. 
Besser so: `<mat-label class="location">` - und dann das Styling in `feedback.component.scss`